src/modules/messages/readable.ts

```diff
@@ -209,7 +209,7 @@ export class AudioData extends Message {
     if (amount === 1) {
       this.command = data.readInt8(12)
     } else if (amount === 4) {
-      this.volumeDuration = data.readUInt32LE(12)
+      this.volumeDuration = data.readFloatLE(12)
```

Comment:
Switched this over to a float, result is always 0.5 which gives the perfect time to defocus the audio for the navigation. Previous value was causing issues as it was being parsed out as 1056964608.